src/Table/index.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {View, Text, ScrollView} from 'react-native';
+import {isEmpty, at} from 'lodash';
+import {Row, Col, TextStrong} from '@indec/react-native-commons';
+
+import RenderComponent from '../mapComponents/index';
+import styles from './styles';
+
+const Table = ({
+    items, attributes, buttons
+}) => (
+    <View style={styles.container}>
+        <Row>
+            {attributes.map(prop => (
+                <Col key={prop.fieldName} flexSize={prop.flexSize}>
+                    <TextStrong style={styles.textItemHead}>{prop.fieldName}</TextStrong>
+                </Col>
+            ))}
+            {!isEmpty(buttons) && buttons.map(button => (
+                <Col key={button.fieldName} flexSize={button.flexSize}>
+                    <TextStrong>{button.fieldName}</TextStrong>
+                </Col>
+            ))}
+        </Row>
+        <View>
+            <ScrollView>
+                {!isEmpty(items) && items.map(item => (
+                    <View
+                        style={[styles.rowDefault]}
+                        key={`dwelling-${item._id}`}
+                    >
+                        <Row>
+                            {attributes.map(prop => (
+                                <Col
+                                    key={`head-${prop.field}${item._id}`}
+                                    flexSize={prop.flexSize}
+                                >
+                                    <Text style={[styles.text]}>
+                                        {at(item, prop.field)}
+                                    </Text>
+                                </Col>
+                            ))}
+                            {!isEmpty(buttons) && buttons.map(button => (!button.childrenButtons ? (
```

Comment:
No me gusta tantos condicionales acá, xq no lo pasamos a una `function` o `arrow function` y reemplazamos esto por `{buttons && this.renderButtons(...)}`